assets/src/dashboard/components/navigationBar/mainNavBar.js

```diff
@@ -23,17 +23,38 @@ import { __ } from '@wordpress/i18n';
  * External dependencies
  */
 import styled from 'styled-components';
-import PropTypes from 'prop-types';
 
 /**
  * Internal dependencies
  */
-import { ReactComponent as WebStoriesLogoSVG } from '../images/logo.svg';
-import { BUTTON_TYPES, DROPDOWN_TYPES, paths } from '../constants';
-import { useConfig } from '../app/config';
-import { useRouteHistory } from '../app/router';
-import Button from './button';
-import Dropdown from './dropdown';
+import { ReactComponent as WebStoriesLogoSVG } from '../../images/logo.svg';
+import { BUTTON_TYPES, DROPDOWN_TYPES, paths } from '../../constants';
+import { useConfig } from '../../app/config';
+import { useRouteHistory } from '../../app/router';
+import Dropdown from '../dropdown';
+import { Nav, ActionLink } from './components';
+
+const PageLinks = styled.div`
+  flex-grow: 1;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 40px;
+  text-align: right;
+
+  @media ${({ theme }) => theme.breakpoint.largeDisplayPhone} {
+    display: none;
+  }
+
+  @media ${({ theme }) => theme.breakpoint.min} {
+    justify-content: flex-end;
+    display: ${({ pathCount = 0 }) => (pathCount > 1 ? 'flex' : 'none')};
```

Comment:
nit: is this something we could just handle with jsx below. like:
```
{paths.length && <PageLinks> ...}
```
Or does it need to be rendered on desktop even if there are no links?

Reply:
I think the parent container uses a `space-between` style of layout, so I believe we want it rendered regardless of linkage to keep the spacing working... I think (this code was also written before and I just moved it).